src/pages/ambassadors.js

```diff
@@ -32,13 +32,20 @@ export default function Ambassadors() {
     return color
   }
 
+  // const ni = [{ name: 'HACKERS', shown: false }, { name: 'ORGANIZERS', shown: false }, { name: 'AMBASSADORS', shown: true }]
+
   return (
     <div>
-      <Navbar />
+      <Navbar currPage="AMBASSADORS" />
       <div className="ambassador-section">
         <div className="header-section">
           <div>
             <div className="horizontal-align">
+              <div className="horizontal-section">
+                <div className="logo-section">
+                  {applicationsOpen ? (<ApplicationsOpenSvg className='open-closed-logo' />) : (<ApplicationsClosedSvg className='open-closed-logo' />)}
+                </div>
+              </div>
```

Comment:
`horizontal-align` previously had `flex-direction: row` in the CSS. Is there a reason why you put this on top and switched it to `flex-direction: row-reverse`? It is visually the same, but having the elements flipped in the HTML and then setting `flex-direction: row-reverse` is a bit more confusing

Reply:
I reversed it because it would've been visually different for mobile view. If it was not reversed, the logo would've been below the "applications are open/closed message" in mobile view.